### DISCUSSION TITLE
Welcome and Introductions

### DISCUSSION THREAD

**Discussion by FrankD412** [llnl/maestrowf#318] [General]
# Maestro Workflow Conductor ([maestrowf](https://pypi.org/project/maestrowf/))

Hello! -- Welcome to the Maestro Workflow discussion board. If you've found your way here and haven't had a change to look at our [README](https://github.com/LLNL/maestrowf/blob/develop/README.md), take a minute to look it over to see how to get started!  At the high level, Maestro is a lightweight workflow tool meant to help researchers and computational scientists spin up multi-step workflows on compute resources (both locally and on HPC clusters).

Whether you're a contributor, user, or software developer/engineer, please feel free to post an introduction here about who you are. We love hearing from our community and hearing about the cool things you all work on!

**Comment by kcathey** [comment]
Hey all! 
During my day job I manage development efforts and operations for a few large monolithic services. I've been programming for over 30 years, from Basic on the Atari 800, to C (in school), to Java, and now Python. I usually work on web services and the like, but I'm up for anything.

For Maestro, mostly I help with the "infrastructure" side of the things, stuff to make the project better to develop.

**Comment by FrankD412** [comment]
Hey everyone! -- I'm a Software Engineer/System Architect and the creator of Maestro. I work full time as a Computer Scientist/Software Engineer primarily in the Computation Biology space focused on cancer research. I use my software skills to help computational biologists/physicists automate their workflow processes and make better use of HPC resources. I have ~10 years of industry experience, working on things ranging from Computer Architecture design all the way up to Software Design and HPC. If you'd like to learn more about me or reach out, check out my [website](https://frankd412.github.io/).
